### ISSUE TITLE
Issue with watched indicator and syncing with trakt

### ISSUE THREAD

**Issue by nived0999** [Tikipeter/repository.tikipeter#461]
i am using Trakt for "watched status provider".I authorized Trakt account without any issue but it does not show any watched status for episodes when i autoplay the next episode.(ie,if i completed first episode and autoplay second episode it doesnot show watched status for the first episode)And also Trakt is not syncing with Fen sometimes.i added episodes to watchlist from trakt.tv but its does'nt shows up in the Fen<my lists<trakt:watch list.i tried restarting kodi to check whether it sync on restarting but it doesnt.

To check whether it trakt api issue i authorised Trakt in kodi TMDB addon but in TMDB trakt watchlist section the movies i watchlisted was there..

the log file is here
https://paste.kodi.tv/yaholigive

when i click toggle watched from the trakt context menu it showed an error.that log file is here
https://paste.kodi.tv/luxohoziki

and also a request....can u add a dialogue to play or cancel the next episode at the end the episode instead of the "still watch check"

kodi version  : 19.0 matrix
FEN version :1.8.49
OS               : Android

**Comment by Tikipeter** [comment]
The error in the second log is from the trakt addon, not Fen.

Because of this, I assume you have the trakt addon installed. If you have the trakt addon marking as watched, then Fen will not mark media as watched. Otherwise they will be marked twice.

If you don't use Kodi's library, then you shouldn't really have a need for the trakt addon. If you do want to keep it installed, then you should stop it marking addon streams as watched. If you do that Fen will see that and start marking things watched

Information from online services (Trakt, TMDb etc) need to be cached by addons so that calls aren't made to their API service every time the info is needed. As such, cached info is not always up to date. Clear the trakt cache and your changes should be reflected.  You can change the cache length time in settings and also make Fen refresh the cache on each Kodi start.

Um, what else have you got here... Oh yeah, Fen doesn't support episodes in Watchlists. Movies or TV Shows only.

Dialog for Next Episode won't be changing. I don't like that netflix like dialog.

**Comment by nived0999** [comment]
I disabled Trakt addon and issues solved..thanks sir..🙂👍

And if i choose FEN as watched status provider will trakt also mark it as watched in trakt.tv?

Or Is there an option to sync all the FEN watch history with trakt when ever we needed...so that we can keep FEN as watch status provider because trakt API is down sometime know..

**Comment by Tikipeter** [comment]
If you have Fen set to use "Trakt" for Watched Status Provider, then Fen will set the marked watched for trakt when meida is played through it. If you also select "Sync Fen Watched Status with Trakt Watched Status", then when Trakt is down, you can temporarily switch Watched Status Provider to "Fen" and it should mimic your current Trakt watched history. Be sure to switch back to "Trakt" when you can, as things watched during this period of Trakt downtime will not be recorded to Trakt. You would need to do this manually when Trakt goes back online.

Basically, if you want to use Trakt, then just select "Trakt" as Watched Status Provider, and select "Sync Fen Watched Status with Trakt Watched Status" below that to keep a local database of your watched history for when Trakt goes down.

**Comment by nived0999** [comment]
Okay..thanks sir...👍
